File: BOJ/Java/단계별로_풀어보기/B_02_조건문/BOJ_14681.java
package 단계별로_풀어보기.B_02_조건문;

/**
 * BOJ 14681 : 사분면 고르기
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class BOJ_14681 {
	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		int X = Integer.parseInt(br.readLine());
		int Y = Integer.parseInt(br.readLine());

		if (X > 0) {
			if (Y > 0) {
				System.out.println(1);
			} else {
				System.out.println(4);
			}
		} else {
			if (Y > 0) {
				System.out.println(2);
			} else {
				System.out.println(3);
			}
		}
	}

}

File: BOJ/CPP/단계별로_풀어보기/02_조건문/BOJ_14681.cpp
/*
    BOJ 14681 : 사분면 고르기

    author : kdgyun
    link https://st-lab.tistory.com
    link https://github.com/kdgyun
    Created by kdgyun
    Copyright © 2021 kdgyun. All rights reserved.
*/

#include <iostream>

using namespace std;

int main(int argc, char const *argv[]) {
    int X, Y;
    cin >> X >> Y;

    cout << (X > 0 ? (Y > 0 ? 1 : 4) : (Y > 0 ? 2 : 3));
    return 0;
}
